test/views/MentionsAndSentiments/test.index.js

```diff
@@ -0,0 +1,131 @@
+                'results':[
+                  {
+                    'key_as_string':'1488412800',
+                    'key':1488412800000,
+                    'matching_results':74,
+                    'aggregations':[
+                      {
+                        'type':'term',
+                        'field':'docSentiment.type',
+                        'results':[
+                          {
+                            'key':'positive',
+                            'matching_results':49
+                          },
+                          {
+                            'key':'negative',
+                            'matching_results':21
+                          },
+                          {
+                            'key':'neutral',
+                            'matching_results':4
+                          }
+                        ]
+                      }
+                    ]
+                  },
+                ]
+              }
+            ]
+          },
+          {
+            'key':'Company Two',
+            'matching_results':465,
+            'aggregations':[
+              {
+                'type':'timeslice',
+                'field':'blekko.chrondate',
+                'interval':'1d',
+                'results':[
+                  {
+                    'key_as_string':'1488326400',
+                    'key':1488326400000,
+                    'matching_results':4,
+                    'aggregations':[
+                      {
+                        'type':'term',
+                        'field':'docSentiment.type',
+                        'results':[
+                          {
+                            'key':'positive',
+                            'matching_results':4
+                          }
+                        ]
+                      }
+                    ]
+                  },
+                  {
+                    'key_as_string':'1488412800',
+                    'key':1488412800000,
+                    'matching_results':43,
+                    'aggregations':[
+                      {
+                        'type':'term',
+                        'field':'docSentiment.type',
+                        'results':[
+                          {
+                            'key':'positive',
+                            'matching_results':1000000
+                          },
+                          {
+                            'key':'negative',
+                            'matching_results':3
+                          },
+                          {
+                            'key':'neutral',
+                            'matching_results':1
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                ]
+              }
+            ]
+          },
+        ]
+      }
+    ]
+  };
+
+  let queryAAA = {
+    date: {
+      from: '20170301',
+      to: '20170501'
```

Comment:
You don't have quotes around the keys here and a space between the key/value but do not follow this convention with the `mentionsAAA` object. I think this convention here is better but the main thing is to be consistent. A good case for a couple lint rules.